### ISSUE TITLE
Multivec track shows horizontal crosshair when mouse pointer is not over multivec track

### ISSUE THREAD

**Issue by alexpreynolds** [higlass/higlass#814]
Using v1.7.2 of higlass, I have a `horizontal-multivec` track along with a few other non-multivec tracks (gene annotations, `horizontal-point`, and chromosome tracks). 

When the mouse pointer moves over a non-multivec track, the multivec track behaves as though the vertical coordinate of the mouse pointer is also on the multivec track, which enables the horizontal line of the crosshair:

![Crosshair](https://user-images.githubusercontent.com/33584/70937208-c115c000-1ff8-11ea-908f-937eea9165b4.gif)

What I would expect is that, when the multivec is part of the top or bottom of a view, it would only show the horizontal crosshair if the pointer is within the bounds of the multivec track. 

(Likewise, if the bug is consistent between orientations, a multivec that is in the left or right portion of a view would not show the vertical crosshair, unless the pointer is within the bounds of the multivec.)

**Comment by keller-mark** [comment]
Hi, are you still experiencing this bug? I tried to reproduce it here http://higlass.io/app/?config=TuwAlswBRLSkikCjBlbZhw but do not see the behavior you experienced in firefox or chrome.

**Comment by alexpreynolds** [comment]
Yes, but I haven't been able to update my local copy of the higlass plugin due to local changes (unrelated to the multivec track).
